notes/transcript15.md

```diff
@@ -213,108 +213,228 @@ Igor: I guess the signature won't signatures won't match but we can do it a shot
+
+Igor: I think that's a good idea in general. So we should probably do some some sort of list.
+
+### 45:00 - 50:00
+
+Eric: I I don't have a thought but I'm wondering so how do we make this decision whether we accept break breaking protocol or not? How do we consider..
+
+Oskar: Yes so we'll have to figure out, I think that for those who been here for it for like a year or year n a half, we had the Swarm process and it's something like this but in a way like similar to the hyphen improvement process or bittorrent improvement or Bitcoin or EIPs or whatever and it says they're having a process like that where there's some form of consensus among some sort of core contributers or whatever it looks like and that will also be its own sort of SIP or whatever you want to call it. Atleast that's how I'm thinking about right now and I think it is a pretty scalable and and and tested process for protocol changes in general and something that's generally used. So that's how I imagined. Specifically how decision will be made or when something will move from draft is always something we can discuss further.
+
+Ricardo: So it would also be possible to option to the users to opt-in this and also other protocol is not using it anymore. But of course this opt-in should be explaining the user what changes and what's why it's better, why  less privacy and etc.
+
+Oskar: So so this would be so this would be more about client implementers and then maybe it's the Status app would choose to expose such options but across I mean it's more amount of Status clients for example like BitTorrent they introduced deep support and all clients have pretty much implemented because it was useful and then you added like a private field to have private trackers and local peer discovery and these types of things and it's like enhancements but it's up to each client how you want to how do you want to deal with it and most clients will likely expose it in some way if you want to use THT and then there's all these considerations and so on. So that's  that in itself could be a local decision for the Status app and it would be separate from the protocol. So the idea is to think of them as two sort of separate animals kind of. That'll be a help in any insights or or thoughts you would like to add to this.
```

Comment:
```suggestion
Oskar: So this would be more about client implementers and then maybe it's the Status app would choose to expose such options but across I mean it's more amount of Status clients for example like BitTorrent they introduced deep support and all clients have pretty much implemented because it was useful and then you added like a private field to have private trackers and local peer discovery and these types of things and it's like enhancements but it's up to each client how you want to how do you want to deal with it and most clients will likely expose it in some way if you want to use THT and then there's all these considerations and so on. So that's  that in itself could be a local decision for the Status app and it would be separate from the protocol. So the idea is to think of them as two sort of separate animals kind of. That'll be a help in any insights or or thoughts you would like to add to this.
```